latency/src/bin/z_ping.rs

```diff
@@ -18,22 +18,21 @@ use std::collections::HashMap;
 use std::time::{Duration, Instant};
 use clap::Parser;
 use zenoh::config::Config;
-use zenoh::net::protocol::core::WhatAmI;
+use zenoh_protocol_core::{WhatAmI, CongestionControl};
 use zenoh::net::protocol::io::reader::{HasReader, Reader};
 use zenoh::net::protocol::io::SplitBuffer;
 use zenoh::prelude::*;
-use zenoh::publication::CongestionControl;
 
 
 #[derive(Debug, Parser)]
 #[clap(name = "z_ping")]
 struct Opt {
-    /// locator(s), e.g. --locator tcp/127.0.0.1:7447,tcp/127.0.0.1:7448
+    /// endpoint(s), e.g. --endpoint tcp/127.0.0.1:7447,tcp/127.0.0.1:7448
     #[clap(short, long)]
-    locator: Option<String>,
+    endpoint: Option<String>,
     
     /// peer, router, or client
```

Comment:
If the possible values only cover "peer" and "client", then the help message shouldn't include "router"?

Reply:
Edited.